_posts/2020-01-21-snowplow-objc-tracker-1.2.0-released/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Snowplow Obj-C Tracker 1.2.0 released"
+title-short: Snowplow Obj-C Tracker 0.9.0
+tags: [snowplow, objc, tracker]
+author: Alex Benini
+category: Releases
+permalink: /blog/2020/01/21/snowplow-objc-tracker-1.2.0-released/
+discourse: false
+---
+
+We are pleased to announce a new release of the [Snowplow Obj-C Tracker][objc-tracker].
+
+On [Version 1.2.0][1.2.0-tag] we fixed few a rare and annoying bugs, but the big news is the support for watchOS. It has been added with the contribution from [Leo Mehlig][leoasana] who updated the tracker watchOS making it more complete and multi-purposeful.
```

Comment:
> who updated the tracker watchOS making it more complete and multi-purposeful.

I'm not sure something can be more multi-purposeful, it can be multi-purpose but I think the objc tracker already was to a certain degree. Change to something like:

> who updated the tracker with watchOS support, allowing it to track events from millions of Apple Watch devices.